Documentation/networking.md

```diff
@@ -1,14 +1,31 @@
+# Networking
+
+`rkt` has two networking options: "host" networking where the apps runs in the root host network namespace or "private" where the apps are allocated a new IP address and a new network namespace.
+
 ## Host (default) mode
 
-By default, `rkt run` will start the pod with host networking. This means that the apps within the pod will share the network stack and the interfaces with the host machine. Since metadata service uses the container IP for indentity, rkt will _not_ register the container with the metadata service in this mode.
+By default, `rkt run` will start the pod with host networking.
+This means that the apps within the pod will share the network stack and the interfaces with the host machine.
+Since metadata service uses the container IP for indentity, rkt will _not_ register the container with the metadata service in this mode.
 
 ## Private networking mode
 
-If `rkt run` is started with `--private-net`, the pod will be executed with its own network stack. By default, Rocket will create a loopback device and a veth device. The veth pair creates a point-to-point link between the container and the host. Rocket will allocate an IPv4 /31 (2 IP addresses) out of 172.16.28.0/24 and assign one IP to each end of the veth pair. It will additionally set a route for metadata service (169.254.169.255/32) and default route in the container namespace. Finally, it will enable IP masquerading on the host to NAT the egress traffic.
+For all of the private networking options the metadata service, launched via `rkt metadatasvc`, must be running.
```

Comment:
This feels a bit out of place, but I guess we can split it out into a metadata service doc later